ocs_ci/ocs/resources/pod.py

```diff
@@ -139,13 +139,14 @@ def exec_cmd_on_pod(self, command, out_yaml_format=True, secrets=None, **kwargs)
             secrets (list): A list of secrets to be masked with asterisks
                 This kwarg is popped in order to not interfere with
                 subprocess.run(**kwargs)
+            timeout (int): timeout for the exec_oc_cmd, defaults to 600 seconds
 
         Returns:
             Munch Obj: This object represents a returned yaml file
         """
         rsh_cmd = f"rsh {self.name} "
         rsh_cmd += command
-        return self.ocp.exec_oc_cmd(rsh_cmd, out_yaml_format, secrets=secrets, **kwargs)
+        return self.ocp.exec_oc_cmd(rsh_cmd, out_yaml_format, secrets=secrets, timeout=timeout, **kwargs)
```

Comment:
Same as above, this line is longer than 79 characters, please reformat it to multiple lines.